config/main.py

```diff
@@ -715,6 +730,12 @@ def _stop_services():
     if asic_type == 'mellanox' and 'pmon' in services_to_stop:
         services_to_stop.remove('pmon')
 
+    disable_services = _get_disabled_services_list()
```

Comment:
This was a typo on my part and now you've copied and pasted it. Sorry. would you mind changing `disable_services` to `disabled_services` wherever it's used as part of this PR, please?